circle.yml

```diff
@@ -1,23 +1,23 @@
+general:
+  artifacts:
+    - "tmp"
+
 machine:
+  environment:
+    ACCESSLINT_MASTER_BRANCH: dev
   ruby:
     version:
       2.3.1
+  node:
+    version: 6.1.0
```

Comment:
The default node version on Circle CI doesn't support features that are required by the `accesslint-cli` node module, so we must specify a more recent version.